main.go

```diff
@@ -79,9 +86,9 @@ func main() {
 	}
 
 	if len(args) == 0 {
-		if runtime.GOOS == "windows" {
+		if *format == "powershell" || (*format == "" && runtime.GOOS == "windows") {
```

Comment:
I think the logic here should be something along the lines of this:

```golang
if *format == "" {
        switch runtime.GOOS{
        case "windows":
                *format = "powershell"
        default:
                *format = "bash"
        }
}

switch *format {
case "powershell":
        printPowershellCredentials(role, creds)
case "bash":
        printBashCredentials(role, creds)
default:
        // Print error about unknown format
}
```

So, if no explicit format is provided, "powershell" will be used on windows, and "bash" will be used on everything else. If an explicit format is provided, then it just uses the provided format.

You can also move the error check into the `default` case at the bottom.